lib/update/cluster/builder.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2019 Gravitational, Inc.
+Copyright 2020 Gravitational, Inc.
```

Comment:
We're not terribly consistent and should really script this at some point, but we use a date range to indicate the content was published and amended over time. So in this case for this particular file, it would be first_publication_year-last_publication_year which would be `Copyright 2019-2020 Gravitational, Inc.`

Example: https://github.com/gravitational/gravity/blob/master/tool/gravity/cli/register.go#L2